src/Client.js

```diff
@@ -78,7 +76,9 @@ function Client(options) {
   this._lastSeen = null;
 
   if (isNodeEnv && opts.keepAlive) {
-    this._keepAliveEnabledAgent = new (isHttps ? https : http).Agent({ keepAlive: true });
+    this._keepAliveEnabledAgent = new (
+      isHttps ? require('https') : require('http')
+    ).Agent({ keepAlive: true });
```

Comment:
I thought the dynamic requires was an issue. 🤔

Reply:
not dynamic requires, just weird (maybe incorrect) syntax using `require` with a variable: https://github.com/fauna/faunadb-js/blob/2.8.1/src/Client.js#L73

no warnings/errors running this through webpack (e.g., `netlify-lambda`)

Reply:
Ahh ok, cool!